test/e2e/upgrade_kubernetes_test.go

```diff
@@ -0,0 +1,205 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	bmov1alpha1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	corev1 "k8s.io/api/core/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/cluster-api/test/framework"
+	"sigs.k8s.io/cluster-api/util/patch"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Kubernetes (version) k8s_upgrade in target nodes [k8s_upgrade]", func() {
+	upgradeKubernetes()
```

Comment:
Would you please add UpgradeKubernetesInputs like https://github.com/metal3-io/cluster-api-provider-metal3/blob/fa6a05594977cdf7511a1e0d2cd8e57b86f378f1/test/e2e/node_reuse.go#L27 so we can use local vars for e2e namespace .. instead of global

Reply:
done